src/main/resources/org/jpeek/metrics/MWE.xsl

```diff
@@ -44,12 +44,14 @@ SOFTWARE.
   </xsl:template>
   <xsl:template match="class">
     <!--
-    @todo #17:30min The current implementation of MWE metric expects information
-     about LDA topics and their probability in skeleton.xml by xpath `class/methods/method/topics/topic`.
-     `topic` items must have two mandatory attributes - `@name` (topic unique identifier) and `@p` (probability of the
-     topic's occurence in the method). Currently this part of information is missing in skeleton.xml.
-     Ensure that the core part of JPeek provides the information about topics and their probability
-     and update the metric in accordance with the provided XML structure.
+    @todo #17:30min To calculate the Maximal Weighted Entropy(MWE)
+     the "topic" is needed. To calculate the topics: a method contains words
+     excluding programming language keywords, stop words and punctuation.
+     So, to calculate the MWE metric each variable's name is a topic.
+     A High Cohesive Class has two characteristics:
+     - High Occupancy (Oi): the main topic is present in all the methods.
+     - High Distribution (Di): all the methods are related to the same topic.
+     For more information check the PDF [lyu09] in the papers.
```

Comment:
@uurl it's not clear from this text what exactly do you want a programmer to do. What is the task?